src/app/core/models/issue.model.ts

```diff
@@ -97,6 +109,10 @@ export class Issue {
       : this.issueOrPr === 'Issue'
       ? Milestone.IssueWithoutMilestone
       : Milestone.PRWithoutMilestone;
+
+    if (githubIssue.reviews) {
+      this.reviews = githubIssue.reviews.map((review) => new PullrequestReview(review));
+    }
```

Comment:
Let's leverage typescript optional chaining for this
```suggestion
    this.reviews = githubIssue.reviews?.map((review) => new PullrequestReview(review));
```

This have the same behavior as the original code.